base/filesystem.jl

```diff
@@ -44,6 +44,8 @@ import .Base:
     skip, stat, unsafe_read, unsafe_write, write, transcode, uv_error,
     rawhandle, OS_HANDLE, INVALID_OS_HANDLE, windowserror
 
+import .Base.RefValue
```

Comment:
Moved the import in the module file, since it's used by both `path.jl` and `file.jl`    (it was indirectly called in `path.jl`  even though the import is in `file.jl`) this should make the intent clear.